Elevel/Domain/Models/Grammar.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Elevel.Domain.Models
+{
+    public class Grammar: BaseDataModel
+    {
+        public Guid QuestionId { get; set; }
+        public Question<Grammar> Question { get; set; }
```

Comment:
what is question and where this class is? And I am not sure that it will be easy for DB to place.  We can discuss it if you want

Reply:
Lets discuss

Reply:
Questions is not implemented entity, Evgeni Kovpak is doing it

Reply:
> what is question and where this class is? And I am not sure that it will be easy for DB to place. We can discuss it if you want
Questions is not implemented entity, Evgeni Kovpak is doing

Reply:
Questions is not implemented entity, Evgeni Kovpak is doing

Reply:
it's bad practice to commit broken build. at least comment that and Evgeny will uncomment if so